.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
+-   repo: https://github.com/awslabs/git-secrets
+    rev: 80230af
+    hooks:
+        -   id: git-secrets
+-   repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
Might be good to include some general-purpose Java linting hooks, e.g. one of [these](https://github.com/gherynos/pre-commit-java).